File: Data Structures & Algorithms/two-integer-sum/submission-1.go
func twoSum(nums []int, target int) []int {
    mp := make(map[int]int)
    ans := make([]int,0)
    for index , value := range nums {
        val := target - value
        if in2 , exists := mp[val]; exists{
            ans = append(ans,in2)
            ans = append(ans,index)
            return ans
        }
        mp[value] = index
    }
    return ans ;
}

File: Data Structures & Algorithms/majority-element/submission-1.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int freq = 0 , ans = -1;
        for(auto i : nums){
            if(freq == 0) ans = i;
            if(ans == i) freq++;
            else freq--;
        }
        return ans;
    }
};